package.json

```diff
@@ -1,24 +1,25 @@
 {
   "name": "business-create-ui",
-  "version": "5.3.1",
+  "version": "5.4.0",
   "private": true,
   "appName": "Create UI",
   "sbcName": "SBC Common Components",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "lint": "eslint . --ext .js,.ts,.vue",
```

Comment:
You can add some extra params here `--ignore-path .gitignore` - not sure if we'd like to do that though

Reply:
That's what .eslintignore is for 😉

Reply:
Sweet, didn't know

Reply:
Ok to resolve this?